python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -877,7 +877,8 @@ def _auto__init__(self, registry=None, unprepare=None):
         self._setDirty()
 
         super(Job, self)._auto__init__()
-        stripProxy(self.info).uuid = str(uuid.uuid4())
+        if self.info is not None:
+            stripProxy(self.info).uuid = str(uuid.uuid4())
```

Comment:
Would it make sense instead to ensure that a job always has a `JobInfo` class in `self.info`?